### PR TITLE
invert sign for era5 radiation variables

### PR DESCRIPTION
**Pull request by Zeitsperre**

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@aulemahal I just realized, this should be named closer to `_switch_direction`.

File: miranda/convert/data/ecmwf_cf_attrs.json [hunk 1]
```diff
@@ -179,6 +189,11 @@
         "era5-single-levels": "J m-2",
         "era5-single-levels-preliminary-back-extension": "J m-2"
       },
+      "_invert_sign": {
+        "era5-land": true,
+        "era5-single-levels": true,
+        "era5-single-levels-preliminary-back-extension": true
```
Comment (aulemahal) [inline miranda/convert/data/ecmwf_cf_attrs.json:195]: I believe it is the opposite!

ERA5 specifies : "The ECMWF convention for vertical fluxes is positive downwards."
And CF conventions for "surface_downwelling_shortwave_flux_in_air", which is listed as `rsds` says : "The sign convention is that "upwelling" is positive upwards and "downwelling" is positive downwards."

**Comment by aulemahal** [comment]
Oh and to "support" my claims, I downloaded `ssr`, `str`, `ssrd` and `strd` from ERA5 and compared my data with `rlds`, `rlus`, `rsds` and  `rsus` from the MRCC5. AFAIU, the ERA5 signs are CF-correct. Thus `rsus = -(ssr - ssrd)`, where the sign change is because `rsus` is upwards.

**Review by aulemahal — APPROVED** [review]
Looks good to me!

**Pull request merged**